### DISCUSSION TITLE
Clarity template limited functionality

### DISCUSSION THREAD

**Discussion by duzenko** [vmware/clarity#6175] [Q&A] [answered]
Clarity seems to have moved its stackblitz templates to beta webcontainers
I don't see any way there to manage dependency versions
Can we have the regular templates as well please?
Regular UI
![image](https://user-images.githubusercontent.com/1481807/126661637-48cc66a6-4c17-4557-8965-1a480a19f52d.png)
Webcontainer beta
![image](https://user-images.githubusercontent.com/1481807/126661704-f522a310-2573-431f-b35f-199a82e5da65.png)

**Comment by coryrylan** [comment, marked as answer]
You can manage the dependencies using the package.json. Once updated then run `npm install` and `npm start` in the terminal.

https://stackblitz.com/edit/clarity-light-theme-v5?file=package.json

**Reply by duzenko** [reply]
While that answers my question about the packages, I'm still wondering why move the templates to the beta system, especially not compatible with Firefox or Safari

**Reply by coryrylan** [reply]
The reason was that stackblitz stopped working for most of our packages with an error `Unexpected token 'export'`. We are in contact with the Stackblitz team but their primary focus is the Web Containers project and have not been able to look into the issue.

https://stackblitz.com/edit/clarity-dark-theme-v5-ksmylj
